docs/guides/server/caching.adoc

```diff
@@ -330,6 +330,24 @@ The `jgroups.fd.port-offset` property defines the offset from the `jgroups.bind.
 
 NOTE: Use `-D<property>=<value` to modify the ports above in your `JAVA_OPTS_APPEND` environment variable or in your CLI command.
 
+[#network-bind-address]
+== Network bind address
+
+To ensure a healthy {project_name} clustering, the network port must be bound on an interface that is accessible from all other nodes of the cluster.
+
+By default, it picks a site local (non-routable) IP address, for example, from the 192.168.0.0 or 10.0.0.0 address range.
+
+To override the address, set the `jgroups.bind.address` property.
+
+NOTE: Use `-D<property>=<value` to modify the ports above in your `JAVA_OPTS_APPEND` environment variable or in your CLI command.
```

Comment:
```suggestion
NOTE: Use `-Djgroups.bind.address=<IP or keyword>` to modify the bind address in your `JAVA_OPTS_APPEND` environment variable or in your CLI command.
```

I'm not sure if you want to extend this doc into an advanced guide on JGroups, but the property also accepts other keywords like `match-interface:eth0` and a list of IPs or keywords to try (`match-interface:eth2,10.5.5.5,match-interface:en.\*,127.0.0.1`). 
See http://jgroups.org/manual5/index.html#Transport

Reply:
For now, let's keep it restricted to IP addresses only. If we see the need arising from user discussions, we might extend it.